### PR TITLE
Updating screenshots and slight wording changes

### PR DESCRIPTION
**Pull request by rr1000**

### CODE REVIEW

File: accessing-your-environment/accessing-the-dashboard.md [hunk 1]
```diff
@@ -6,16 +6,16 @@ title: Accessing the dashboard
...
-Login to the dashboard using the credentials you created earlier.
+Sign in to the dashboard using the credentials you created earlier.
 
 You will see a screen that looks something like this:
 
-![View Deployed App](/assets/img/pics/32.view.deployed.app.png)
+![View Deployed App](http://cdn2.dropmark.com/45280/4ce8f48eaae63fd58ee8fff263b1173e61111da4/db_archive-list.png)
 
-You can click on View (right next to the environment name) or the Overview link to the right to see a summary screen of the containers that have been deployed and their status.
+To see a summary of your environments services you can click "Overview" in the navigation at the bottom the main environment panel.
```
Comment (anthonypleshek) [inline accessing-your-environment/accessing-the-dashboard.md]: "at the bottom `of` the main environment"

File: deploying-your-first-app/define-connections-between-your-containers.md [hunk 2]
```diff
@@ -6,32 +6,32 @@ title: Define connections between your containers
...
+Now that you have defined the various services within your environment, the next step is to tell the Catalyze PaaS how you would like them to be wired up. We ask you to make these connections explicitly as to not make assumptions on your behalf.
 
-The convention we follow is data centric i.e. the store or source of data (e.g. database) **provides** data to the **consumer** of data (e.g. the application). So in the example that we've been working through, the PostgreSQL service provides data to the Ruby service. So you would select **db01** from the Data Provider drop down and **App01** from the Data Consumer dropdown.
+The convention we follow is data centric i.e. the store or source of data (e.g. database) **provides** data to the **consumer** of data (e.g. the application). So in the example that we've been working through, the PostgreSQL service provides data to the Ruby service. So you would select **db01** from the Data Provider drop down and **App01** from the Data Consumer dropdown. (**Note:** The diagram on the right is fully interactive. It add a connection simply drag and arrow from a data provider to a data consumer)
```
Comment (anthonypleshek) [inline deploying-your-first-app/define-connections-between-your-containers.md]: "(**Note:** The diagram on the right is fully interactive. To add a connection, simply drag an arrow from a data provider to a data consumer.)"

File: deploying-your-first-app/review-your-settings.md [hunk 3]
```diff
@@ -10,16 +10,12 @@ title: Review your settings
 
 Once all the information has been entered, you will be provided with this screen that summarizes it all.
 
-![Summary Screen](/assets/img/pics/30.review.png)
+![Summary Screen](http://cdn2.dropmark.com/45280/c1d964d99a5fff66d33835089c7b84a33f193fd8/db_review.png)
 
-Please spend some time and carefully review your settings and data that you have entered. While it is possible to delete your app after it has been deployed in case of errors, I'm sure you would prefer not spending a bunch of time trying to figure out why something is not working. If there are errors at this point, you can always hit the yellow **Back** button. Please don't use the browser back button. While it mostly works, there are cases with specific browsers that might cause some issues.
+Please spend some time carefully reviewing your settings and the data that you have entered. While it is possible to delete your environment after it has been deployed in the event of an error, I'm sure the majority of users would prefer not spending a considerable amount of time attempting to debug the issue.
```
Comment (anthonypleshek) [inline deploying-your-first-app/review-your-settings.md]: Should reword to remove the `I'm` from this.

Maybe something like "in the event of an error, the majority of users will not want to spend a considerable amount of time"

Comment (mohan2020) [inline deploying-your-first-app/review-your-settings.md]: Leave it. Makes it a bit personal. Documentation doesn't have to be dry.

**Pull request merged**